indigo/bulk_creator.py

```diff
@@ -382,19 +381,27 @@ def link_publication_document(self, work, info):
         pub_doc.save()
 
     def link_commencement(self, work, info):
-        # if the work is `commenced_by` something, try linking it
-        # make a task if this fails
-        title = info['commenced_by']
-        work = info['work']
-        commencing_work = self.find_work_by_title(title)
-        if not commencing_work:
-            return self.create_task(work, info, task_type='link-commencement')
-
-        work.commencing_work = commencing_work
-        try:
-            work.save_with_revision(self.user)
-        except ValidationError:
-            self.create_task(work, info, task_type='link-commencement')
+        # if the work has commencement details, try linking it
+        # make a task if a `commenced_by` title is given but not found
+        commencing_work = None
+        date = info.get('commencement_date')
+        all_provisions = work.all_provisions()
```

Comment:
this will probably be empty since no points in time exist, yet. That's okay, just double checking.